src/psyclone/psyir/transformations/acc_kernels_trans.py

```diff
@@ -148,6 +161,10 @@ def validate(self, nodes, options=None):
                 "GOcean InvokeSchedules")
         super().validate(node_list, options)
 
+        # Any errors below this point will optionally log the resason, which
```

Comment:
s/resason/reason/

Reply:
This has gone following changes on master.